src/SignalR/clients/cpp/src/signalrclient/connection_impl.cpp

```diff
@@ -23,22 +24,30 @@ namespace signalr
 
     std::shared_ptr<connection_impl> connection_impl::create(const std::string& url, trace_level trace_level, const std::shared_ptr<log_writer>& log_writer)
     {
-        return connection_impl::create(url, trace_level, log_writer, std::make_unique<web_request_factory>(), std::make_unique<transport_factory>());
+        return connection_impl::create(url, trace_level, log_writer, nullptr, std::make_unique<transport_factory>());
     }
 
     std::shared_ptr<connection_impl> connection_impl::create(const std::string& url, trace_level trace_level, const std::shared_ptr<log_writer>& log_writer,
-        std::unique_ptr<web_request_factory> web_request_factory, std::unique_ptr<transport_factory> transport_factory)
+        std::unique_ptr<http_client> http_client, std::unique_ptr<transport_factory> transport_factory)
```

Comment:
Nah, https://github.com/aspnet/AspNetCore/pull/8420/files#diff-94c5397b9dcb8a27dcc5dd7ca166abe6R42 😛

Reply:
Instead of passing nullptr here: https://github.com/aspnet/AspNetCore/pull/8420/files#diff-94c5397b9dcb8a27dcc5dd7ca166abe6R27, pass in https://github.com/aspnet/AspNetCore/pull/8420/files#diff-94c5397b9dcb8a27dcc5dd7ca166abe6R48